mr/dependencies.lock

```diff
@@ -1,4 +1,365 @@
 {
+    "allProcessors": {
```

Comment:
This `allProcessors` block doesn't look right. @rdblue I ran ` ./gradlew :iceberg-mr:generateLock` and then ` ./gradlew :iceberg-mr:saveLock` to generate this file. Did I miss something?

Reply:
This is also added in #1086, so I think it is the result of a change to config that wasn't reflected in the existing lock files. I wouldn't worry about it too much, unless it's causing issues. This configuration should just be internal to Gradle.

Reply:
OK, I'll just look away because it does seem to be working ;)